### PR TITLE
Feature #17 extra fields

### PR DESCRIPTION
**Pull request by AlejandroVI99**
# Description
What did you implemented/Why did you implemented this:
 - Have added name and lastname fields in the Student registration page
 - Have added name, lastname, about, tags, summary, object fields in the Teacher registration page
 - Have gereted the controllers for Student and Teacher from devise
 - Have edited the controllers of Student and Teacher to permit new attributes
 - Have deleted old Student controller and views that were not use 
  
  ## Screenshots 
Screenshot from 2021-08-19 14-14-36
Screenshot from 2021-08-19 14-14-51
Screenshot from 2021-08-19 14-15-31
Screenshot from 2021-08-19 14-15-42

### CODE REVIEW

File: app/views/students/registrations/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :lastname %><br />
+    <%= f.text_field :lastname, autofocus: true, autocomplete: "lastname" %>
+  </div>
```
Comment (oramasangel) [inline app/views/students/registrations/edit.html.erb:42]: you should remove those <br/>, cause when you apply the styles, use a padding in each div or in the ruby code tag to set that space between the elements

**Review by oramasangel — APPROVED** [review]
LGTM

**Review by xFerchoVJ — DISMISSED** [review]
LGTM

File: app/views/students/registrations/edit.html.erb (hunk 1 above)
Comment (AlejandroVI99) [reply app/views/students/registrations/edit.html.erb:42]: Ok! understood!

**Review by YilKanda — APPROVED** [review]
Just as a recommendation, try to make the PR title more specific, since you indicate that you added extra fields, but do not explain where.

**Review by SampabloC — APPROVED** [review]
LGTM

**Review by xFerchoVJ — APPROVED** [review]
LGTM

**Pull request merged**